### ISSUE TITLE
displayLocationInDecimal missing in WSDL

### ISSUE THREAD

**Issue by ttomaszewski** [forcedotcom/wsdl#8]
The Metadata WSDL is missing the following tag element, which is causing the `readMetadata` method to fail when using the SOAP Metadata API.

```
com.sforce.ws.ConnectionException: unable to find end tag at:  START_TAG seen ...</fullName><displayLocationInDecimal>... @305:5283
```

This was reported in December in Trailblazer - https://trailblazer.salesforce.com/issues_view?id=a1p4V000002ML1YQAW&title=attribute-displaylocationindecimal-is-not-available-in-metadata-wsdl

**Comment by gbockus-sf** [comment]
Howdy.  Is [this](https://github.com/forcedotcom/wsdl/blob/9a23edc2388692867f6eb23acc822ba91283270c/resources/metadata.wsdl#L6367) the element which is missing when looking at the org WSDL?

**Comment by ttomaszewski** [comment]
> Howdy. Is [this](https://github.com/forcedotcom/wsdl/blob/9a23edc2388692867f6eb23acc822ba91283270c/resources/metadata.wsdl#L6367) the element which is missing when looking at the org WSDL?

@gbockus-sf Yes, however, when I download the Metadata WSDL via `Setup > API` from my Org, this element is missing...

**Comment by gbockus-sf** [comment]
@ttomaszewski understood. In order to get an answer on the in org WSLD you'll have to open a support investigations with Salesforce.  Here we take the values in source an provide it in the repo, but don't have any insight into how the org generates the WSDL shown when accessed via that route.
